### PR TITLE
Feature/update tailwind theme

### PR DESCRIPTION
**Pull request by RPinkha**
## Describe your changes

tailwind.config.js has been updated to reflect the mocks in [figma](https://www.figma.com/design/TPZy9TvTUPyC1RV1nuY0bq/Event-Solutions?node-id=0-1&t=x5savO0zQwWMCNj5-0).

Things changed includes:

    Colors, backgrounds

    Fonts

    Breakpoints

Note: Spacing was not added. Tailwind comes with default spacing, so when building a component consider looking at tailwind docs to find default spacing values. If a value isn't present, the config needs to be edited to add the value. 

## Issue ticket number and link

EES2

https://tripleten-apiary.atlassian.net/browse/EES-2

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { faWheatAlt } from "@fortawesome/free-solid-svg-icons";
```
Comment (XDRO) [inline tailwind.config.js]: Unnecessary import as it is not being used.

File: tailwind.config.js [hunk 2]
```diff
@@ -6,43 +7,28 @@ export default {
   mode: "jit",
   purge: ["./public/**/*.html", "./src/**/*.{js,jsx,ts,tsx,vue}"],
```
Comment (XDRO) [inline tailwind.config.js]: The `purge` option is deprecated in favor of the `content` option, options have changed in Tailwind CSS v3.0.
image This issue is leading to this warning 
image

File: tailwind.config.js [hunk 3]
```diff
@@ -6,43 +7,28 @@ export default {
   mode: "jit",
```
Comment (XDRO) [inline tailwind.config.js]: You can safely remove 'jit'
image

**Review by XDRO — DISMISSED** [review]
There are several outdated use cases in this file. One of which cause the `@talwind` rule to be unknown.

**Comment by RPinkha** [comment]
> There are several outdated use cases in this file. One of which cause the `@talwind` rule to be unknown.

So the `@tailwind` rule seems to be unknown not because of the config file, but because of CSS lint.
Here is a stack overflow thread:
https://stackoverflow.com/questions/76776910/unknown-at-rule-tailwind-cssunknownatrules-tailwind-error

**Review by michaelabudde — APPROVED** [review]
I have reviewed all of the values and verified that they are correct according to the figma.

File: tailwind.config.js [hunk 4]
```diff
@@ -3,46 +3,30 @@ import HeadlessTailwindPlugin from "@headlessui/tailwindcss";
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ["./index.html", "./src/**/*.{html,js,jsx,ts,tsx}"],
-  mode: "jit",
-  purge: ["./public/**/*.html", "./src/**/*.{js,jsx,ts,tsx,vue}"],
+  content: ["./public/**/*.html", "./src/**/*.{js,jsx,ts,tsx,vue}"],
```
Comment (tomershvueli) [inline tailwind.config.js]: Let's combine these separate `content` arrays into 1 array, the line below should cover all cases that these 2 arrays contain: 
`content: ["./index.html", "./public/**/*.html", "./src/**/*.{html,js,jsx,ts,tsx,vue}"]`

**Review by tomershvueli — APPROVED** [review]
Looks great!

**Pull request merged**